### PR TITLE
Pass relative URLs for static files through django.contrib.staticfiles (fixes #111)

### LINKED ISSUE #111: Better support for static files of plugins
Thanks for a great app!

In ``leaflet/__init__.py``, this statement:

    urls[i] = urljoin(settings.STATIC_URL, url)

does not take the settings for Django's static files backend into account. This means md5 hashes, S3 storage, etc. will all cause the plugins to fail.

A solution would be to have Django ``static files`` generate the url, and replace the code above with:

    from django.contrib.staticfiles.templatetags.staticfiles import static
    urls[i] = static(url)

See also: http://stackoverflow.com/questions/11721818/django-get-the-static-files-url-in-view

I could do a pull request, but since it's only one line I figured this was the fastest solution with a proper explanation of the problem. I hope you can use this.

### PR DESCRIPTION
**Pull request by dyve**

### CODE REVIEW

File: leaflet/__init__.py [hunk 1]
```diff
@@ -182,7 +183,8 @@ def _normalize_plugins_config():
                     # absolute URL or a URL starting at root
                     pass
                 else:
-                    urls[i] = urljoin(settings.STATIC_URL, url)
```
Comment (leplatrem) [inline leaflet/__init__.py]: Error on travis: imported but no used :)

Comment (dyve) [inline leaflet/__init__.py]: I'll add a new PR
@leplatrem I suggest you add instructions on how to locally run the same test as Travis -- python quicktest.py leaflet --db=postgres (which I had to figure out for myself) passed perfectly. Contributing without knowing the tests is no fun.

Comment (dyve) [inline leaflet/__init__.py]: Maybe add 'setup.py test' as default test script, it's not hard

**Comment by dyve** [comment]
I removed the obsolete import of url join. Please add instructions to run local tests that will also pass Travis.

**Comment by leplatrem** [comment]
I created a dedicated issue for the docs, to prevent blocking the merge of this one :)

Thanks a lot for your patience and contribution :)

**Pull request merged**